PowerShell/JumpCloud Module/Public/Commands/Get-JCCommand.ps1

```diff
@@ -1,10 +1,34 @@
 
 Function Get-JCCommand ()
 {
-    [CmdletBinding(DefaultParameterSetName = 'ReturnAll')]
+    [CmdletBinding(DefaultParameterSetName = 'SearchFilter')]
 
     param
     (
+
+    #### NEW PARAMS
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'The command to execute on the server.')]
+        [String]$command,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Name of the command')]
+        [String]$name,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Command Type')]
+        [string]$commandType,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Launch Type')]
+        [string]$launchType,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Listens To')]
+        [string]$listensTo,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Schedule')]
+        [string]$schedule,
```

Comment:
I found one issue with this parameter but I'm tempted to remove this parameter because I doubt it's going to be super helpful. 

If you search for a command that has a chron trigger of '0 */5 * * * *' the asterisks are never sent to the command:
<img width="775" alt="Screen Shot 2022-06-21 at 8 24 38 PM" src="https://user-images.githubusercontent.com/54448601/174929815-c631d5f0-8401-494a-aeba-f2a19b7d192a.png">

Let's not worry about a validated set here but I think we should specify that schedules are in cron tab format.
<img width="414" alt="Screen Shot 2022-06-21 at 8 18 01 PM" src="https://user-images.githubusercontent.com/54448601/174929096-39c8a964-e97c-46d9-bcb0-661f234d5e68.png">

Thoughts on just removing this parameter?

Reply:
@kmaranionjc  to remove